Tests/FunctionalTests/NetworkResponseHandlerFunctionalTests.swift

```diff
@@ -576,9 +576,9 @@ class NetworkResponseHandlerFunctionalTests: FunctionalTestBase {
             "            ]}],\n" +
             "      \"errors\": [" +
             "        {\n" +
-            "          \"code\": \"personalization:2003\",\n" +
-            "          \"namespace\": \"personalization\",\n" +
-            "          \"message\": \"Failed to process personalization event\"\n" +
+            "          \"status\": 2003,\n" +
```

Comment:
Status now represents the http status code, so this value is not possible. Let's update these tests with some accurate examples.

Reply:
typo whoops.